examples/schemas/global.json

```diff
@@ -0,0 +1,39 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"type": "object",
+	"definitions": {
+		"time_span": {
+			"type": "string",
+			"pattern": "^[0-9]+:[0-5]?[0-9]$"
+		}
+	},
+	"properties": {
+		"name": {
+			"type": "string"
+		},
+		"staffId": {
+			"type": "integer",
+			"minimum": 1000000,
```

Comment:
Do you know for sure that the staffId may not begin with leading zeros (or may be shorter respectively)?

Reply:
No, I don't. Thinking about it, we should remove the range assertion.